src/app/components/Search/SearchSuggestionsLinksForNoResults.tsx

```diff
@@ -31,7 +32,7 @@ export const SearchSuggestionsLinksForNoResults: FC<Props> = ({ scope }) => {
   const { isMobile } = useScreenSize()
   const { suggestedBlock, suggestedTransaction, suggestedAccount, suggestedTokenFragment } =
     (scope?.network && scope?.layer && searchSuggestionTerms[scope.network][scope.layer]) ??
-    searchSuggestionTerms['mainnet']['sapphire']!
```

Comment:
I somewhat prefer string literals :shrug:

Reply:
I am fine with either method, but up to now we had 95% constants, and 5% string literals. At least now it's uniform...

Reply:
> I somewhat prefer string literals 🤷

See https://github.com/oasisprotocol/explorer/pull/2016 🤷🏻